source/_integrations/zwave_js.markdown

```diff
@@ -20,56 +20,81 @@ ha_codeowners:
 
-## Configuration
+4. The Z-Wave JS integration in Home Assistant. This integration connects to the Z-Wave JS Server to retrieve the info from your Z-Wave network and turns it into Home Assistant devices and entities.
 
-This integration can be configured using the integrations in the
-Home Assistant frontend.
+### Running the Z-Wave JS Server
 
-Menu: **Configuration** -> **Integrations**.
+As stated in the above requirements, you need to run the Z-Wave JS Server somewhere in your network. There are multiple ways to run this server, we'll explain the most common ways below:
 
-Click on the `+` sign to add an integration and click on **Z-Wave JS**.
-After completing the configuration flow, the Z-Wave JS integration will be
-available.
+#### 1. The official Z-Wave JS add-on, available from the add-on store
 
-### Secure network key
+This is the recommended approach if you're running the default version of Home Assistant, including the Supervisor and you do not require any advanced configuration/options but just want your Z-Wave devices available to control within Home Assistant.
 
-The secure network key is set in the settings for zwave-js-server and
-not in the integration configuration.
+#### 2. The Z-Wave JS to MQTT add-on installed available from the community add-on store
 
-## Current Limitations
+This is the recommended approach if you're running the default version of Home Assistant, including the Supervisor and you like to have maximum control over your Z-Wave network and its nodes.
 
-As this integration is still in the early stages there are some important limitations to be aware of.
+Despite what the name suggests, you can actually run this addon without MQTT enabled. In that case, it will provide you with a full-fledged, attractive and feature-complete UI to manage your Z-Wave nodes and settings.
 
-- Garage door controllers (Barrier Operator CC) are not currently supported.
-- Configuration of Z-Wave nodes and/or configuration with the Home Assistant UI is currently not yet implemented. You will need to use another tool, such as [zwavejs2mqtt](https://github.com/zwave-js/zwavejs2mqtt), to manage device configuration.
-- Polling is currently not supported but may be added in a later release.
-- Support for setting configuration parameters through service calls is currently not supported but may be added in a later release.
-- There currently is no migration path from any of the other Z-Wave implementations in Home Assistant. Your Z-Wave network is however stored on your stick so migrating will only require you to redo your device and entity naming.
+The Z-Wave JS server is **included** in Zwave JS to MQTT, you enable it on the Z-Wave tab within the UI. If you install the add-on from the community repo, this will be done for you automatically.
```

Comment:
```suggestion
The Z-Wave JS server is **included** in Zwave JS to MQTT. You enable it on the Z-Wave tab within the UI. If you install the add-on from the community repo, this will be done for you automatically.
```